de.peeeq.wurstscript/src/main/java/de/peeeq/wurstio/utils/W3Utils.java

```diff
@@ -6,77 +6,90 @@
 import net.moonlightflower.wc3libs.port.NotFoundException;
 import net.moonlightflower.wc3libs.port.Orient;
 import net.moonlightflower.wc3libs.port.StdGameExeFinder;
-import net.moonlightflower.wc3libs.port.mac.MacGameVersionFinder;
+import net.moonlightflower.wc3libs.port.StdGameVersionFinder;
 import net.moonlightflower.wc3libs.port.win.WinGameExeFinder;
 
 import java.io.File;
 import java.io.IOException;
+import java.util.Optional;
 
 public class W3Utils {
```

Comment:
Is there any circumstance where this class not being a singleton is required?

Reply:
It wasn't a singleton before, it was static